dcos/package.py

```diff
@@ -58,10 +58,13 @@ def install_app(pkg, revision, init_client, options, app_id):
     init_desc = pkg.marathon_json(revision, options)
 
     if app_id is not None:
-        logger.debug('Setting app ID to "%s" (was "%s")',
-                     app_id,
-                     init_desc['id'])
-        init_desc['id'] = app_id
+        if not isinstance(init_desc, list):
+            logger.debug('Setting app ID to "%s" (was "%s")',
+                         app_id,
+                         init_desc['id'])
+            init_desc['id'] = app_id
+        else:
+            logger.debug('Not setting app ID for multi app json')
```

Comment:
Let's think about this some more. It is possible that we can just remove `--app-id` from the `dcos package install` command. I am not sure if anyone uses this feature. Let me think about it.

To me, it sounds good to add support for multiple app to the `dcos marathon app` commands. I am worry about using this in the `dcos package install` command. I need to understand what is driving this change. Let's meet on Monday so I can better understand the user story.